src/api/public/apidocs-new/paths/person.yaml

```diff
@@ -0,0 +1,33 @@
+post:
+  summary: Executes a command on the person endpoint.
+  description: >-
+    Allows executing command on the person endpoint.
+    As of now, the only command allowed is 'register', even if it says that the only allowed command is 'change_password'.
```

Comment:
See [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/person_controller.rb#L29) for details.

Reply:
I know that this PR is only about documenting the current state of this endpoint, but I would be in favor of changing this line in the command endpoint:

https://github.com/openSUSE/open-build-service/blob/cc9879e0541feb708cbcf053d0433eb8d1197dc5/src/api/app/controllers/person_controller.rb#L33

It would be `raise UnknownCommandError, "Allowed commands are 'register'"`. We could get rid of this confusing part `even if it says that the only allowed command is 'change_password'` in the documentation and code.

Reply:
Yes, do the right thing. This doesn't cost anything...

Reply:
Changed. The error message will be fixed [here](https://github.com/openSUSE/open-build-service/pull/10666)